py/desispec/brick.py

```diff
@@ -75,28 +75,30 @@ def __init__(self, bricksize=0.5):
 
     def brickname(self, ra, dec):
         """Return string name of brick that contains (ra, dec) [degrees]
-        
+
         Args:
             ra (float) : Right Ascension in degrees
             dec (float) : Declination in degrees
-            
+
         Returns:
             brick name string
         """
         inra, indec = ra, dec
         dec = np.atleast_1d(dec)
         ra = np.atleast_1d(ra)
         irow = ((dec+90.0+self._bricksize/2)/self._bricksize).astype(int)
-        names = list()
-        for i in range(len(ra)):
-            ncol = self._ncol_per_row[irow[i]]
-            j = int(ra[i]/360 * ncol)
-            names.append(self._brickname[irow[i]][j])
+
+        ncol = self._ncol_per_row[irow]
+        jj = (ra/360.0 * ncol).astype(int)
+        names = np.empty(len(ra), dtype='U8')
```

Comment:
Thanks, @weaverba137.  For the record, I didn't write this function.  However, I think it's OK for ```brickname``` to be hard-coded to 8 characters, which gives four significant figures in RA and three significant figures in Declination (with a p/m to indicate the sign of the declination).  What's nice about this is that it works well with the 0.25x0.25 deg2 bricks currently in use by the imaging surveys, although it also works with the 0.5x0.5 deg2 bricks we're using in the mocks.  We'll never go smaller than 0.25 deg on a side and even with 1 deg bricks it means we'll be carrying around a few extra bytes -- no big deal.